Contrib/mmpa/test_rfrag.py

```diff
@@ -0,0 +1,25 @@
+#
+#  Copyright (C) 2017 greg Landrum
+#
+#   @@ All Rights Reserved  @@
+#
+import unittest, subprocess, os
+from rdkit import RDConfig
+
+
+class TestCase(unittest.TestCase):
+  def test1Github1406(self):
+    with open('data/simple.smi') as inf:
+      p = subprocess.run(('python', 'rfrag.py'),stdin=inf,stdout=subprocess.PIPE)
+    self.assertFalse(p.returncode)
+    self.assertEqual(p.stdout,b'''c1ccccc1,benzene,,
+Cc1ccccc1,toluene,,C[*:1].c1ccc(cc1)[*:1]
+''')
+
+
+if __name__ == '__main__':
+  import sys
+  if sys.hexversion >= 0x3050000:
+    unittest.main()
+  else:
+    print('Python >=3.5 required to run tests')
```

Comment:
I guess this limitation is because of the comparison on line 15. The following might work on 2.7 and 3.x
```
stdoutData = p.stdout
if sys.hexversion >= 0x3050000:  # bytes to string
    stdoutData = stdoutData.decode()
self.assertEqual(stdoutData, '''c1ccccc1,benzene,,
Cc1ccccc1,toluene,,C[*:1].c1ccc(cc1)[*:1]
''')
```